pycbc/inference/option_utils.py

```diff
@@ -581,6 +581,12 @@ def results_from_cli(opts, load_samples=True, **kwargs):
     else:
         samples = None
 
+    # why is this not included in the above transforms??
```

Comment:
Make a transform for ``eta`` if you're using it, Don't include this code block here.